File: Iniciante/1117.py
contador = 0
somador = 0

while True:
  nota = float(input())

  if nota >= 0 and nota <= 10:
    contador += 1
    somador += nota

    if contador == 2:
      print("media = {:.2f}".format(somador/contador))
      break

  else:
    print("nota invalida")

File: Iniciante/1117.js
lines = lines.map((e) => Number(e));

let contador = 0;
let somador = 0;

for (let i = 0; i < lines.length; i++) {
  if (lines[i] >= 0 && lines[i] <= 10) {
    contador++;
    somador += lines[i];
    if (contador == 2) {
      console.log("media = " + (somador/contador).toFixed(2));
      break;
    }
  } else {
    console.log("nota invalida")
  }
}
